### DISCUSSION TITLE
Force dnscrypt-proxy to use a single DoH server

### DISCUSSION THREAD

**Discussion by pmcarrion** [DNSCrypt/dnscrypt-proxy#1952] [Q&A] [answered]
I just installed dnscrypt-proxy and dnscrypt-proxy switcher on my Mac.

Everything works as expected, but I want to use a specific DoH URL and disable all the other servers that are built-in.

Can someone advise how to do this?

Thanks.

**Comment by jedisct1** [comment]
Set `server_names` to the list of servers you want to use.

See https://github.com/DNSCrypt/dnscrypt-proxy/wiki/Configuration and the rest of the documentation :)

**Comment by pmcarrion** [comment, marked as answer]
I was finally able to force DNScrypt-Proxy to exclusively use my DoH URL, while still using Little Snitch.
There's a macOS bug that disables DoH profiles if Little Snitch is used.

Follow these steps:

1) Open dnscrypt-proxy.toml from your dnscrypt-proxy folder in a text editor that shows line numbers (I used TextMate with the TOML plug-in).

2) Comment out lines 671 to 685.

3) Add these lines at the end of the file (they should be lines 859 and 860).

```
[static.'DoH Profile Name']
stamp = 'sdns://DoH-SDNS'
```

Go here to create the [SNDS](https://dnscrypt.info/stamps/).

Once you input this information your SDNS will be shown at the top right side of the website.

Copy the Stamp (SDNS) in `stamp = 'sdns://DoH-SDNS'` of line 860

Reboot
